thirdparties-extension/fr.opensagres.poi.xwpf.converter.pdf.itext5/src/main/java/fr/opensagres/poi/xwpf/converter/pdf/internal/elements/StylableDocument.java

```diff
@@ -94,7 +96,32 @@ public void addElement( Element element )
         }
         text.addElement( element );
         StylableDocumentSection.getCell( layoutTable, colIdx ).getColumn().addElement( element );
-        simulateText();
+        try
+        {
+            if ( ColumnText.hasMoreText( text.go( true ) ) )

```

Comment:
Just one question, what about performance?

Reply:
That part that you are commenting on (if ( ColumnText.hasMoreText( text.go( true ) ) )) is identical to what is done in the simulateText(); that this replaces. The only difference here is that it doesn't assign to the local variable res and doesn't call that extra method which if anything would improve performance. So I am not entirely sure what you are asking here.